Modules/Utils/Sources/PocketCastsUtils/Feature Flags/FeatureFlag.swift

```diff
@@ -93,7 +93,7 @@ public enum FeatureFlag: String, CaseIterable {
     }
 
     private var shouldEnableSyncedSettings: Bool {
-        false
+        true
```

Comment:
Shall we do like we did before? https://github.com/Automattic/pocket-casts-ios/commit/2858b76ce31c2c97314a4a5d01861fb166e4c716

Reply:
Yeah, sure. It provides a backstop against a future release, I guess.